src/components/application_manager/include/application_manager/commands/mobile/set_app_icon_request.h

```diff
@@ -87,9 +87,11 @@ class SetAppIconRequest : public CommandRequestImpl {
    * @brief Remove oldest icons
    * @param storage Path to icons storage
    * @param icons_amount Amount of icons to be deleted
+   * @param is_correct_removed_out Checks, if removing of icons was correct
    */
   void RemoveOldestIcons(const std::string& storage,
-                         const uint32_t icons_amount) const;
+                         const uint32_t icons_amount,
+                         bool& is_correct_removed_out) const;
```

Comment:
@VVeremjova, output parameters should have `out_` prefix.
It also rather be good to avoid there usage of output parameter if it's possible.